### PR TITLE
Documentation review

### PR DESCRIPTION
**Pull request by jooola**
With this PR is sync the docs with the latest commit i made on the website AND I fix some docs after i made a review !

I think that after this is merged we could merge your pr to the documentation and submodule...

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ layout: docs
...
-- [Exe package (Windows)]({{ site.baseurl }}/docs/install/exe)
-- [Pkg package (macOS)]({{ site.baseurl }}/docs/install/pkg)
-- [Build from source]({{ site.baseurl }}/docs/install/source)
+Here you can pick your [installation guide](/docs/install) for:
+- [WAR package (Tomcat)](/docs/install/war)
+- [WAR package (Standalone)](/docs/install/war-standalone)
+- [Deb package (Debian / Ubuntu)](/docs/install/deb)
+- [Rpm package (Red Hat / Fedora)](/docs/install/rpm)
+- [Exe package (Windows)](/docs/install/exe)
+- [Pkg package (macOS)](/docs/install/pkg)
```
Comment (muff1nman) [inline install.md]: These don't apply so shouldnt we remove them?

File: install/deb.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ permalink: /docs/install/deb/
...
-- [A JDK installation, 1.8.x series of OpenJDK or Oracle JDK 8+ should work.]({{ site.baseurl }}/docs/install/prerequisites)
+- [A JDK installation, 1.8.x series of OpenJDK or Oracle JDK 8+ should work.](/docs/install/prerequisites)
 
 #### Install Libresonic package
 
-Download the latest Libresonic .deb package from the [download page]({{ site.baseurl }}/download), or with the command below:
+Download the latest Libresonic .deb package from the [download page](/download), or with the command below:
 
 ```
 wget {{ site.repo }}/libresonic-v{{ site.stable_version }}.deb
```
Comment (muff1nman) [inline install/deb.md]: Same as above, shouldn't we remove these files?

File: install.md (hunk 1 above)
Comment (jooola) [inline install.md]: We can comment them out yes but i want to keep them. All the the package page are ready to be written !
Use the website directly you will understand what is really going on !
https://libresonic.github.io/docs/

File: install/deb.md (hunk 2 above)
Comment (jooola) [inline install/deb.md]: Again if we can comment them out i'll prefer ! ( Just like the [download page] (https://raw.githubusercontent.com/Libresonic/Libresonic.github.io/master/pages/download.md) )

I'll comment them out give me 5 minutes

Comment (jooola) [inline install/deb.md]: actually i made a mistake sorry, those files should not exist ! I'll remove them ! You can't reach them using the navigation bar on website

**Comment by muff1nman** [comment]
Getting closer 👍

**Comment by jooola** [comment]
So i would like to keep the DEB EXE PKG pages here since this is a base to build a proper documentation. we will remove them once we have set our build targets !

Some docs aren't up to date but they are warning messages before them ( Build from source docs ) I just keep them so we don't need to rewrite them totally !

**Pull request merged**